Makefile

```diff
@@ -164,6 +164,20 @@ go-run:
 				--operator-namespace=default --namespace= \
 				--auto-install-webhooks=false
 
+go-debug:
+	@(cd cmd &&	AUTO_PORT_FORWARD=true dlv debug \
```

Comment:
Was it intentional to add this? (I'm not against it).
Is it mandatory to run `dlv debug` from the `cmd/` directory or `dlv debug ./cmd/main.go manager` could work?
Then, I wonder if we should put `go run` in a variable to make it possible to replace it by `dlv debug`.

Reply:
If Delve is not run in the same directory as `main` it requires the package to be specified. Unfortunately, it seems to have problems parsing both `./cmd` and `github.com/elastic/cloud-on-k8s/cmd` apparently because we don't have any Go files at the package root. I didn't spend too much time on understanding why but I am going to do so later.

Unfortunately, the command execution is slightly different between `go-debug` and `go-run` so there's no straightforward way to reuse the targets either.